pyramid/config/views.py

```diff
@@ -1427,27 +1459,30 @@ def forbidden(request):
         view will be invoked.  Unlike
         :meth:`pyramid.config.Configurator.add_view`, this method will raise
         an exception if passed ``name``, ``permission``, ``context``,
-        ``for_``, or ``http_cache`` keyword arguments.  These argument values
-        make no sense in the context of a forbidden view.
+        ``for_`` or ``require_csrf``  keyword arguments. These argument values
+        make no sense in the context of a forbidden :term:`exception view`.
 
         .. versionadded:: 1.3
+
+        .. versionchanged:: 1.8
+
+           The view is created using ``exception_only=True``.
         """
         for arg in (
-            'name', 'permission', 'context', 'for_', 'http_cache',
-            'require_csrf',
+            'name', 'permission', 'context', 'for_', 'require_csrf',
```

Comment:
I couldn't think of a reason why it was prevented versus any other view deriver option... There's nothing that enables http caching automatically and so the option is only there if people really want to opt-in to use it... and if they want to, I don't see a reason to stop them.